DependencyInjection/Compiler/ResourcePass.php

```diff
@@ -62,6 +58,23 @@ public function process(ContainerBuilder $container)
     }
 
     /**
+     * Adds a list of operations.
+     *
+     * @param ContainerBuilder $container
+     * @param string           $serviceId
+     * @param array            $methods
+     * @param bool             $collection
+     *
+     * @return Reference[]
+     */
+    private function createOperations(ContainerBuilder $container, $serviceId, $methods, $collection)
+    {
+        return array_map(function ($method) use ($container, $serviceId, $collection) {
```

Comment:
Using methods as key should make the call more explicit and more performant (even if it not so important here because this method should not be called at runtime).